### ISSUE TITLE
Recommended parameters for Takara Bio SMARTer Human TCR v2 do not work. --index-file FALSE needs to also be set

### ISSUE THREAD

**Issue by jowkar** [nf-core/airrflow#196]
### Description of the bug

The instructions to run airrflow 2.3.0 according the "Takara Bio SMARTer Human TCR v2" section at https://nf-co.re/airrflow/2.3.0/usage#dt-oligo-rt-and-5race-pcr are as follows:

```
nextflow run nf-core/airrflow -profile docker \
--input samplesheet.tsv \
--library_generation_method dt_5p_race_umi \
--cprimers CPrimers.fasta \
--race_linker linker.fasta \
--umi_length 12 \
--umi_position R2 \
--cprimer_start 5 \
--cprimer_position R1 \
--outdir ./results
```

However, when using these settings on my data, the pipeline rapidly exits with the following error:

```
Cannot invoke method and() on null object

 -- Check script '/home/ubuntu/.nextflow/assets/nf-core/airrflow/./workflows/bcellmagic.nf' at line: 100 or see '.nextflow.log' file for more details
```

Examining line 100 in bcellmagic.nf, as suggested, gives the following:

```
if (params.index_file & params.umi_position == 'R2') {exit 1, "Please do not set `--umi_position` option if index file with UMIs is provided."}
```

Setting the parameter `--index_file FALSE` solves this issue and the pipeline successfully launches. This suggests the pipeline expects a default value of FALSE to be present for params.index_file, but that this parameter value is in fact empty.

**Comment by ggabernet** [comment]
This will be solved by #238

**Comment by ggabernet** [comment]
Closing this issue as this has been solved in release 3.0
